pkgs/os-specific/linux/framework-laptop-kmod/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, linuxPackages
+, kernel
+}:
+
+stdenv.mkDerivation rec {
+  pname = "framework-laptop-kmod";
+  version = "unstable-2023-12-03";
+
+  src = builtins.fetchGit {
```

Comment:
use fetchFromGitHub

that should fix ci (nix-prefetch-github DHowett framework-laptop-kmod --nix --rev ...)

Reply:
Thx